services/skipchain/msgs.go

```diff
@@ -33,6 +36,21 @@ func init() {
 // deny a SkipBlock.
 type VerifierID uuid.UUID
 
+var verifiers map[VerifierID]bftcosi.VerificationFunction
+var verifiersMutex sync.Mutex
+
+// RegisterVerification stores the verification in a map and will
+// call it whenever a verification needs to be done.
+func VerificationRegistration(v VerifierID, f bftcosi.VerificationFunction) error {
```

Comment:
Inconsistency between names here `RegisterVerification` vs `VerificationRegistration`. (I have a preference for the former)